Week2/mehmet.js

```diff
@@ -0,0 +1,167 @@
+// 1. Write a console.log statement saying "Hello World!" for each language that you know.
+
+console.log(`Merhaba Dünya!
+Hello World!
+Ciao Mondo!
+Hallo Wereld!`);
+
+// 2. Consider the following code:
+
+console.log(`I'm awesome`);
+
+// 3. Declare a variable x and initialize it with an integer, using these exact steps:
+// 3.1
+let x; // Declaration and initialization
+
+// 3.2
+console.log(`the value of my variable x will be: undefined`);
+
+// 3.3
+console.log(x);
+
+// 3.4
+x = 9;
+
+// 3.5
+console.log(`the value of my x is 9`);
+
+// 3.6
+console.log(x);
+
+// 4
+let y = `house number 80`;
```

Comment:
> Unless you need to include a variable inside a string or have a multiline strings, I would suggest that you use either single quotes `'` or double quotes `"` for creating strings 🙂

But didn't we talked at the lecture to use that (`..`) kind of single quotes at the lecture (The one below the esc?

Reply:
It could have been my mistake then. My apologies. The idea is that we should use `back-ticks` if we want to have variables inside a string or use multiline strings. More info can be found here - https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Template_literals

For other strings, the standard is usually to use single quotes or double quotes.